docker/docker-compose-with-azblob.yml

```diff
@@ -19,7 +19,7 @@ services:
                  --compute-opts=\" \
                     --config-path /risingwave.toml \
                     --listen-addr 0.0.0.0:5688 \
-                    --prometheus-listener-addr 0.0.0.0:1222 \
+                    --prometheus-listener-addr 0.0.0.0:1250 \
```

Comment:
Does this work? From my understanding, these `--prometheus-listener-addr` doesn't have any actually effect because only the Meta node's.

Reply:
The same for `--health-check-listener-addr`. It would be better if we prevent `--prometheus-listener-addr` and `--health-check-listener-addr` to be specified in standalone mode.

Reply:
These seem to be a separate refactor, out of scope for this PR. Tracked here https://github.com/risingwavelabs/risingwave/issues/12086.

Yeah I think we can remove them and/or ban them in standalone mode.

Reply:
+1 for https://github.com/risingwavelabs/risingwave/issues/12086